x/ecocredit/server/ormutil/db_test.go

```diff
@@ -0,0 +1,62 @@
+package ormutil
+
+import (
+	"testing"
+
+	"github.com/cosmos/cosmos-sdk/orm/model/ormdb"
+	"github.com/cosmos/cosmos-sdk/store"
+	"github.com/cosmos/cosmos-sdk/store/types"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	ecocreditv1beta1 "github.com/regen-network/regen-ledger/api/regen/ecocredit/v1beta1"
+	"github.com/stretchr/testify/require"
+	"github.com/tendermint/tendermint/libs/log"
+	tmproto "github.com/tendermint/tendermint/proto/tendermint/types"
+	dbm "github.com/tendermint/tm-db"
+	"google.golang.org/protobuf/reflect/protoreflect"
+)
+
+func sdkContextForStoreKey(key *types.KVStoreKey) sdk.Context {
+	db := dbm.NewMemDB()
+	cms := store.NewCommitMultiStore(db)
+	cms.MountStoreWithDB(key, sdk.StoreTypeIAVL, db)
+	err := cms.LoadLatestVersion()
+	if err != nil {
+		panic(err)
+	}
+	return sdk.NewContext(cms, tmproto.Header{}, false, log.NewNopLogger())
+}
```

Comment:
I have an impression that I'm  seeing this construction in multiple places (eg in server tests, maybe in other modules as well). Let's maybe move to the root?

Reply:
what do you mean by root?

Reply:
btw this code already lives in `types/` on master but i moved it here to not have to deal with other breaking changes in `types/`